packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -29,6 +34,48 @@
   private final FlutterWebViewClient flutterWebViewClient;
   private final Handler platformThreadHandler;
 
+  // Verifies that a url opened by `Window.open` has a secure url.
+  private class FlutterWebChromeClient extends WebChromeClient {
+    @Override
+    public boolean onCreateWindow(
+        final WebView view, boolean isDialog, boolean isUserGesture, Message resultMsg) {
+      final WebViewClient webViewClient =
```

Comment:
Not sure I understand this part, it seems like we're creating a new WebViewClient type and delegate some of the calls to the existing flutterWebViewClient, do we not want all functionality implemented by FlutterWebViewClient to work for the new WebView as well? (e.g should we not just use FlutterWebViewClient as the client for the new webview?)

Reply:
I assumed that we only wanted the new `WebView` to filter out non `https`/`http` calls and load any secure url. I'm not sure what else we would need it to do? Im assuming you're talking about returning `WebResourceError`s and `onPageStarted`/`onPageFinished` callbacks. Won't our FlutterWebViewClient receive these after we call `loadUrl`?